[2024-04-10T09:56:41.625+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-10T09:56:41.718+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trigger_airbyte_job_example_sheet.airbyte_check_sync scheduled__2024-04-09T00:00:00+00:00 [queued]>
[2024-04-10T09:56:41.756+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trigger_airbyte_job_example_sheet.airbyte_check_sync scheduled__2024-04-09T00:00:00+00:00 [queued]>
[2024-04-10T09:56:41.757+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-10T09:56:41.825+0000] {taskinstance.py:2327} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync> on 2024-04-09 00:00:00+00:00
[2024-04-10T09:56:41.853+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1021) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-10T09:56:41.856+0000] {standard_task_runner.py:63} INFO - Started process 1025 to run task
[2024-04-10T09:56:41.863+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'trigger_airbyte_job_example_sheet', 'airbyte_check_sync', 'scheduled__2024-04-09T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/dag_aribyte_ex.py', '--cfg-path', '/tmp/tmp33islinn']
[2024-04-10T09:56:41.871+0000] {standard_task_runner.py:91} INFO - Job 23: Subtask airbyte_check_sync
[2024-04-10T09:56:42.148+0000] {task_command.py:426} INFO - Running <TaskInstance: trigger_airbyte_job_example_sheet.airbyte_check_sync scheduled__2024-04-09T00:00:00+00:00 [running]> on host ff62e46d2b31
[2024-04-10T09:56:42.521+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='trigger_airbyte_job_example_sheet' AIRFLOW_CTX_TASK_ID='airbyte_check_sync' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-09T00:00:00+00:00'
[2024-04-10T09:56:42.526+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-10T09:56:42.527+0000] {baseoperator.py:399} WARNING - AirbyteJobSensor.execute cannot be called outside TaskInstance!
[2024-04-10T09:56:42.612+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-10T09:56:42.669+0000] {airbyte.py:104} INFO - Waiting for job 140 to complete.
[2024-04-10T09:57:42.677+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-10T09:57:42.710+0000] {airbyte.py:99} INFO - Job 140 completed successfully.
[2024-04-10T09:57:42.711+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-04-10T09:57:42.711+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-10T09:57:42.723+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=trigger_airbyte_job_example_sheet, task_id=airbyte_check_sync, execution_date=20240409T000000, start_date=20240410T095641, end_date=20240410T095742
[2024-04-10T09:57:42.781+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-10T09:57:42.801+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-10T09:57:42.804+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
